spec/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--require spec/setup.js
+--recursive
```

Comment:
Can we pass them in the mocha setup as an obj? I don't really see a
difference in having it as one big thing here vs one big thing in
package.json

On Thu, Apr 23, 2015, 4:29 PM Eric Baer <notifications@github.com> wrote:

> In spec/mocha.opts
> <https://github.com/patrickkettner/krabby/pull/21#discussion_r29000097>:
>
> > @@ -0,0 +1,2 @@
> > +--require spec/setup.js
> > +--recursive
>
> I'd rather use a file to encapsulate what is happening than building up a
> big 1 liner in the package.json. Mocha has a LOT of options
> <http://mochajs.org/#usage> so it can get big but I could go either way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/patrickkettner/krabby/pull/21/files#r29000097>.
>